src/command_modules/azure-cli-iotcentral/azure/cli/command_modules/iotcentral/custom.py

```diff
@@ -14,7 +14,7 @@
 logger = get_logger(__name__)
 
 
-def iotcentral_app_create(cmd, client, app_name, resource_group_name, subdomain, sku="F1", location=None):
+def iotcentral_app_create(cmd, client, app_name, resource_group_name, subdomain, sku="F1", location=None, template=None):
```

Comment:
As part of these changes we should remove all references to the `F1` sku which is going away. My guess is that we will need to start failing API calls with sending `F1` at some point. //cc @zzcodework

Reply:
Are we specifically not supporting F1 in the CLI? I thought we were going to continue to support the 7 day trial app?

Reply:
the 7 day free trial cannot be created through the CLI
S1 apps will get five free devices, so you can think of S1 as a combination of F1 and S1, which is why F1 is going away.